galgebra/ga.py

```diff
@@ -698,15 +696,27 @@ def sdop(self, *args, **kwargs):
             DeprecationWarning, stacklevel=2)
         return dop.Sdop(*args, **kwargs)
 
+    @property
+    def lt_coords(self):
+        # galgebra 0.5.0
+        warnings.warn(
+            "`ga.lt_coords` is deprecated, use the identical `ga.coords`.",
+            DeprecationWarning, stacklevel=2)
+        return self.coords
+
+    @property
+    def lt_x(self):
+        # galgebra 0.5.0
+        warnings.warn(
+            "`ga.lt_x` is deprecated, use the identical `ga.coords_vec`.",
+            DeprecationWarning, stacklevel=2)
+        return self.coord_vec
+
     def lt(self, *args, **kwargs):
         """
         Instanciate and return a linear transformation for this, 'self',
         geometric algebra.
         """
-        if not self._lt_flg:
-            self._lt_flg = True
-            self.lt_coords, self.lt_x = lt.Lt.setup(ga=self)
-
```

Comment:
Any compensation for deleting them?

Reply:
Not sure what you mean. Are you saying I should just remove them?

As written in this patch, these attributes still exist and behave as they did before, but now emit `DeprecationWarning`.